src/components/Title/__snapshots__/Title.spec.jsx.snap

```diff
@@ -1,23 +1,23 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Title should display children 1`] = `
-<ForwardRef
+<ForwardRef(MainTitle)
   className="Title"
   ellipsis={true}
   tag="h1"
 >
   content
-</ForwardRef>
+</ForwardRef(MainTitle)>
```

Comment:
Nit, but isn't it strange that these snapshots have changed?

Reply:
Yes, I suspect that at some point a newer version of cozy-ui was used to make the screenshot. THis should not be part of the PR. I removed this change.